notebooks/neural_tangent_kernels.ipynb

```diff
@@ -117,13 +117,15 @@
    "id": "62bc6b5a-31fa-411e-8069-e6c1f6d05248",
    "metadata": {},
    "source": [
-    "## Compute the NTK: method 1\n",
+    "## Compute the NTK: method 1 (Jacobian contraction)\n",
     "\n",
-    "We're ready to compute the empirical NTK. The empirical NTK for two data points `x1` and `x2` is defined as an inner product between the Jacobian of the model evaluated at `x1` and the Jacobian of the model evaluated at `x2`:\n",
+    "We're ready to compute the empirical NTK. The empirical NTK for two data points $x_1$ and $x_2$ is defined as the outer product between the Jacobian of the model evaluated at $x_1$ and the Jacobian of the model evaluated at $x_2$:\n",
```

Comment:
Thanks for the review Richard!

You're right that it's not a proper outer product, however I think it's also not an inner product since it produces a matrix (`O x O`), not a scalar output. I originally made the choice in favor of outer product name since it notationally looked like an outer product (`J J^T`) vs an inner product (`J^T J`). How about using just "matrix product" then (just pushed) to be more accurate?